app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,10 @@
   <%= yield :meta_tags %>
   <%= yield :meta_section %>
 
+  <% if should_track_mouse_movements? %>
```

Comment:
I need to remove this conditional, but first `govuk_ab_testing` needs to allow for it.